README.md

```diff
@@ -99,3 +99,18 @@ npm test
 ```
 
 Please feel free to send me a PR or open an [Issue](https://github.com/mustafar/parrot/issues).
+
+# OpenApi 3 Alpha
+
+Make the following changes in swagger-express-middleware
+- in /lib/helpers/util.js, use
+```
+util.isOpenApiRequest = function (req) {
+  // If req.openapi.operation is set, then so are req.openapi.api and req.openapi.path
+  return req.openapi !== undefined && req.openapi.operation !== undefined;
+};
+```
+- add a call to `next()` in the following middleware
+  - validate-request/parse-params.js
+  - validate-request/validate-content-length.js
+  - validate-request/validate-request-body.js
```

Comment:
these are the bugs i was talking about. i'll send them a PR, and hopefully they merge and release a new alpha. if not, I'll figure out a way to bundle in the swagger-middleware in some other way. lmk if you have suggestions